src/Components/Results/Results.css

```diff
@@ -0,0 +1,100 @@
+.results {
+  background-color: rgb(20, 18, 17);
```

Comment:
any reason you chose to use the `rgb` rather than the hex?

Reply:
not really, i was using the digital color meter that comes with macs to get the color before it was updated in the wireframe, and that uses rgb by default